charts/node-analyzer/CHANGELOG.md

```diff
@@ -3,6 +3,11 @@
 ## Change Log
 
 This file documents all notable changes to Sysdig Node Analyzer Helm Chart. The release numbering uses [semantic versioning](http://semver.org).
+## 1.7.31
+### Minor changes:
+* BenchmarkRunner
+    * bump to version 1.1.0.2
+        * bump linux bench to 1.1.0.1
```

Comment:
nit: isn't this an unnecessary level of indentation?

Reply:
it's a nested dependency. I'll just remove it.